README.md

```diff
@@ -94,12 +103,14 @@ If you get this error message:
 It means the exporter is unable to connect to DB2, however it doesn't know why. To fix this error, please ensure the following:
 
 - Verify that the DSN being used by the exporter is correct for the instance/database of DB2 being monitored.
-- Verify that DB2 is running and all of it's communication protocols are activated.
+- Verify that DB2 is running and all of its communication protocols are activated.
```

Comment:
I had to [google this](https://www.scribbr.com/commonly-confused-words/its-vs-its/) to see why it was changed. I would have argued that `it's` is correct, but apparently I am wrong. 😂